src/Microsoft.Framework.PackageManager/NuGet/Core/Configuration/Settings.cs

```diff
@@ -172,15 +169,11 @@ public string ConfigFilePath
             if (configFileName == null)
             {
                 // load %AppData%\NuGet\NuGet.config
-#if DNX451
-                var appDataPath = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
-#else
-                var appDataPath = Environment.GetEnvironmentVariable("APPDATA");
-#endif
-                if (!String.IsNullOrEmpty(appDataPath))
+                var userSettingsDir = DnuEnvironment.GetFolderPath(DnuFolderPath.UserSettingsDirectory);
```

Comment:
`NuGet.Settings.LoadMachineWideSettings(root)` requires you pass `CommonAppData` as its `root` argument. Note that it requires `CommonAppData`, not `CommonAppData/NuGet/Config` because this method append `/NuGet/Config` to root by itself. I am following the same convention here.